src/WebInterface/Presentation/Http/View/layout/forms.html.twig

```diff
@@ -0,0 +1,23 @@
+{% use "bootstrap_5_layout.html.twig" %}
+
+{# Don't apply label_attr to labels for radio buttons. #}
+{%- block choice_label -%}
+    {% if expanded is defined and expanded %}
+        <legend class="form-label{{ required ? ' required' }}">{{ label }}</legend>
+    {% else %}
+        {{- parent() -}}
+    {% endif %}
+{%- endblock choice_label %}
+
+{%- block choice_widget_expanded -%}
```

Comment:
Add a comment about the why. In this case forcing `required` to `false` to not have `*` appended to the label.